src/sanitizer.js

```diff
@@ -186,6 +190,16 @@ export function sanitizeHtml(html) {
         }
         return;
       }
+      const bindAttribsIndices = {};
+      // Special handling for attributes for amp-bind which are formatted as
+      // [attr]. The brackets are restored at the end of this function.
+      for (let i = 0; i < attribs.length; i += 2) {
+        const attr = attribs[i];
+        if (attr && attr[0] == '[' && attr[attr.length - 1] == ']') {
+          bindAttribsIndices[i] = true;
+          attribs[i] = attr.slice(1, -1);
+        }
+      }
       if (BLACKLISTED_TAGS[tagName]) {
         ignore++;
       } else if (tagName.indexOf('amp-') != 0) {
```

Comment:
Hey guys! 😃  There's a balance we can achieve here. Including a couple small cleanups is a low-overhead way to improve code health (and Justin is our resident code health hero), while more than "a couple" can pollute history and make rollbacks trickier (as Kevin mentioned). The [Code Review Dev Guide](https://engdoc.corp.google.com/eng/doc/devguide/coding/review/developer/small-cls.md?cl=head#refactoring) has more info about this.

IMO this shouldn't *block* the PR, but it would only take a couple seconds while you're in the file. And Justin did ask nicely. 😄